tests/main/abort/task.yaml

```diff
@@ -1,5 +1,8 @@
 summary: Check change abort
 
+# Disabled for Fedora until test case is properly ported
+systems: [-fedora-*]
```

Comment:
Indeed, and same for all the cases below. It also means we don't need all those comments, which is nice. Simply enable for fedora one-by-one, until we finally don't need the disabling at the suite level anymore and can take all of them off.